examples/synology/nebula.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+SCRIPT="PATH_TO_CONFIG" # example /volume1/nebula/config.yml
+SUBNET="YOUR SUBNET IP" # example 10.1.0.0/24
+
+PIDFILE="PATH_TO_PID" # example - /volume1/nebula/nebula.pid
+LOGFILE="PATH_TO_LOG" # example - /volume1/nebula/nebula.log
+
+status() {
+  if [ -f $PIDFILE ]; then
+    echo 'Service running' >&2
```

Comment:
We should probably verify that this is not a stale pidfile via something like `kill -0 $(cat $PIDFILE)` here too.